lib/index.ts

```diff
@@ -134,11 +140,17 @@ export async function validate(
             const jsonLines = componentsDefinitionContent.split('\n');
 
             validateSchema.errors.forEach((error) => {
+                if (!componentsDefinitionSourcePointers) {
```

Comment:
This is not part of fixing eslint errors.

Reply:
True, but since the pointers are now typed it the compiler complaints they could be `null`. Therefore added this check.